### PR TITLE
Update README for TPC

### PR DESCRIPTION
**Pull request by kawakami-masaki0**
## Pull Request Description:
- Update README for TPC
  - Add example of `get_target_platform_capabilities_sdsp` API
  - Modify link
  
Note: CICD errors occured because MCTWrapper verification repair is not yet supported.

## Checklist before requesting a review:
- [x] I set the appropriate labels on the pull request.
- [ ] I have added/updated the release note draft (if necessary).
- [ ] I have updated the documentation to reflect my changes (if necessary).
- [ ] All function and files are well documented. 
- [ ] All function and classes have type hints.
- [ ] There is a licenses in all file.
- [ ] The function and variable names are informative. 
- [ ] I have checked for code duplications.
- [ ] I have added new unittest (if necessary).

### CODE REVIEW

File: model_compression_toolkit/target_platform_capabilities/README.md [hunk 1]
```diff
@@ -13,41 +13,52 @@ in some operator for its weights/activations, fusing patterns, etc.)
...
+[TFLite](./tpc_models/tflite_tpc/v1_0/tpc.py) and [QNNPACK](./tpc_models/qnnpack_tpc/v1_0/tpc.py) models were created similarly and were used to create two TPCs: One for Keras TPC and one for PyTorch TPC (for each model, this 8 in total).
 
 ## Usage
 
-The simplest way to initiate a TPC and use it in MCT is by using the function [get_target_platform_capabilities](https://sonysemiconductorsolutions.github.io/mct-model-optimization/api/api_docs/methods/get_target_platform_capabilities.html#ug-get-target-platform-capabilities).
+The simplest way to initiate a TPC and use it in MCT is by using the function [get_target_platform_capabilities](https://sonysemiconductorsolutions.github.io/mct-model-optimization/api/api_docs/methods/get_target_platform_capabilities.html#ug-get-target-platform-capabilities). 
+
+This function gets a TPC object matching the tpc version and device type. For IMX500, the version can be set to '1.0', '4.0', or '5.0'. For TFLite and QNNPACK, this can only be set to '1.0'.
```
Comment (yt0705) [inline model_compression_toolkit/target_platform_capabilities/README.md]: To ensure future compatibility, please avoid limiting your statements to specific versions like v1-v5.
For example: “As the number of supported versions will continue to grow, please refer to the README (Supported Versions)” + include a URL link.

File: model_compression_toolkit/target_platform_capabilities/README.md [hunk 2]
```diff
@@ -13,41 +13,52 @@ in some operator for its weights/activations, fusing patterns, etc.)
...
+tpc = mct.get_target_platform_capabilities(tpc_version='1.0', device_type='imx500')
+
+# Apply MCT on TensorFlow pre-trained model using the TPC.
+quantized_model, quantization_info = mct.ptq.keras_post_training_quantization(in_model=pretrained_model, # Replace with your pretrained model
+                                                                              representative_data_gen=dataset, # Replace with your representative dataset
+                                                                              target_platform_capabilities=tpc)
 ```
 
-Similarly, you can retrieve IMX500, TFLite and QNNPACK target-platform models for Keras and PyTorch frameworks.
+You can also get a TPC for IMX500 using the function [get_target_platform_capabilities_sdsp](../../docs/api/api_docs/methods/get_target_platform_capabilities_sdsp.html#ug-get-target-platform-capabilities_sdsp) that specifies the sdsp converter version. The version can be set to '3.14', '3.16', or '3.17'.
```
Comment (yt0705) [inline model_compression_toolkit/target_platform_capabilities/README.md]: To ensure future compatibility, please avoid limiting your statements to specific versions like 3.14-3.17.
For example: “As the number of supported versions will continue to grow, please refer to the README (Supported Versions)” + include a URL link.

**Review by yt0705 — CHANGES_REQUESTED** [review]

File: model_compression_toolkit/target_platform_capabilities/README.md (hunk 1 above)
Comment (kawakami-masaki0) [reply model_compression_toolkit/target_platform_capabilities/README.md]: Fix to add link for supported versions.

File: model_compression_toolkit/target_platform_capabilities/README.md (hunk 2 above)
Comment (kawakami-masaki0) [reply model_compression_toolkit/target_platform_capabilities/README.md]: Fix to add link for supported versions.

File: model_compression_toolkit/target_platform_capabilities/README.md [hunk 3]
```diff
@@ -13,41 +13,52 @@ in some operator for its weights/activations, fusing patterns, etc.)
...
 - [QNNPACK](https://github.com/pytorch/QNNPACK)
 
 The default target-platform model is [IMX500](https://developer.sony.com/develop/imx500/), quantizes activations using 8 bits with power-of-two thresholds for 
 activations and symmetric threshold for weights.
 For mixed-precision quantization it uses either 2, 4, or 8 bits for quantizing the operators.
-One may view the full default target-platform model and its parameters [here](./tpc_models/imx500_tpc/v1/tpc.py).
+One may view the full default target-platform model and its parameters [here](./tpc_models/imx500_tpc/v1_0/tpc.py).
 
-[TFLite](./tpc_models/tflite_tpc/v1/tpc.py) and [QNNPACK](./tpc_models/qnnpack_tpc/v1/tpc.py) models were created similarly and were used to create two TPCs: One for Keras TPC and one for PyTorch TPC (for each model, this 8 in total).
+[TFLite](./tpc_models/tflite_tpc/v1_0/tpc.py) and [QNNPACK](./tpc_models/qnnpack_tpc/v1_0/tpc.py) models were created similarly and were used to create two TPCs: One for Keras TPC and one for PyTorch TPC (for each model, this 8 in total).
```
Comment (yt0705) [inline model_compression_toolkit/target_platform_capabilities/README.md]: >and were used to create two TPCs: One for Keras TPC and one for PyTorch TPC (for each model, this 8 in total).

Delete as it conflicts with the latest conditions.

Comment (kawakami-masaki0) [reply model_compression_toolkit/target_platform_capabilities/README.md]: Delete unnecessary statements
